app/gui2/src/components/GraphEditor/widgets/WidgetSelfIcon.vue

```diff
@@ -26,14 +26,8 @@ export const widgetDefinition = defineWidget(
 
 <template>
   <SvgIcon
-    class="WidgetSelfIcon icon nodeCategoryIcon"
+    class="WidgetSelfIcon icon nodeCategoryIcon r-24"
```

Comment:
How does `r-24` work? I don't see any definition.

Reply:
It interacts with the port widgets, avoiding adding extra margins around already rounded elements. That allows the icon's port to be rendered as a circle.